### ISSUE TITLE
efficientnet pretrained model URL is empty

### ISSUE THREAD

**Issue by hjlee9182** [rwightman/gen-efficientnet-pytorch#53]
When I use this code

model = geffnet.create_model(
        'efficientnet_b5',
        num_classes=1000,
        in_chans=3,
        pretrained=True,
        exportable=True)

![image](https://user-images.githubusercontent.com/46337218/98530683-9bee0800-22c2-11eb-9846-85942f51bfd7.png)

But this error occured.

efficientnet b0~b3 => okay

efficientnet b4~b7 => URL is empty.

How to solved this problem?

**Comment by linhduongtuan** [comment]
Ross has not trained efficientnet_b5 by himself yet. You can try using tf_efficientnet_b5. Good luck


Linh

**Comment by hjlee9182** [comment]
@linhduongtuan 
You mean tensorflow hub efficientnet?

**Comment by linhduongtuan** [comment]
Not really, he converts/ports TF weights to compatible PT weights.
Linh

**Comment by linhduongtuan** [comment]
1/ I don’t really know. You can look more closely eyes in his repos
2/ instead of using model = create_model(‘efficientnet_b5’, pretrained=True), you should use model = create_model(’tf_efficientnet_b5’, pretrained=True). To know where is a weight link, you can go into timm —> models —> efficientnet.py —> url for them
 3/ Yes, you will use transfer learning with the pretrained weight from Google which is ported to PT
Linh

**Comment by hjlee9182** [comment]
Oh, I understand clearly.
I have one more question.
I see tf_efficientnet is more accurately better than efficientnet.

Did you know the reason that?

**Comment by rwightman** [comment]
thanks @linhduongtuan 

@hjlee9182 all models with the `tf_` prefix were converted from original Tensorflow weights, and are available here https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet ... numerous different papers are covered so quite a few different sets of weights. The models without the prefix were trained in PyTorch by me, I've only had resources to train the smaller ones. The models do differ in the padding used. You can research SAME padding in PyTorch if you are curious why that's relevant.
